build.sbt

```diff
@@ -75,3 +75,12 @@ addCommandAlias(
     " -rff current.throughput.prof.csv" +
     " -prof stack;"
 )
+
+val generateMasterFileTask =
+  taskKey[Unit]("If the master file has not been created, current is copied. This will happen just the first time.")
+generateMasterFileTask := {
+  if (!file("bench/master.avgtime.csv").exists())
+    IO.copy(Seq((file("bench/current.avgtime.csv"), file("bench/master.avgtime.csv"))))
+}
```

Comment:
**suggestion**: No need to do part of this PR, but, could we move this logic to sbt-hood? It seems to be something needed in every project, isn't it?

Reply:
We probably want to do not fail in the hood part instead and set the PR comment to something like: "No other benchmarks results found to compare too. Skipping results". I think that makes more sense for the users than comparing with the same file. Thoughts?